tests/model/model_controller/DefaultModelController.cc

```diff
@@ -260,3 +260,78 @@ TEST_F(DefaultModelControllerTest, GetSpecificSolidTasks) {
     EXPECT_EQ(result.AccessResult()[i], expected[i]);
   }
 }
+
+TEST_F(DefaultModelControllerTest, MustAddLabel) {
+  auto task = task_factory_.GetNextTask();
```

Comment:
might as well be in SetUp

Reply:
This is only used in several tests in this test suite.